README.md

```diff
@@ -1,2 +1,152 @@
 # php-swaggerize-fastroute-library
 A library to automatically create FastRoute routes based on swagger JSON documentation
+# Install
+To install with composer:
+```
+composer require iadvize/php-swaggerize-fastroute-library
+```
+
+# Generate route File (FastRoute compatible)
+
+```
+vendor/bin/swaggerize swagger:scan path/to/swagger/json controllers\namespace [--routeFile=route/file/path]
+```
+
+# Dispatch generated file or simply use cache
+
+You can then use FastRoute cached dispatcher to use generated file or directly use a cache dispatcher (file will be generated at first call).
+
+```PHP
+<?php
+
+require '/path/to/vendor/autoload.php';
+
+$lumenOperationParser = new \Iadvize\SwaggerizeFastRoute\OperationParser\LumenControllerOperationParser('Controllers\\Namespace\\');
+
+$dispatcher = FastRoute\cachedDispatcher(function(FastRoute\RouteCollector $r, ['cacheFile' => 'route/file/path']) {
+    \Iadvize\SwaggerizeFastRoute\addRoutes(
+        'https://raw.githubusercontent.com/wordnik/swagger-spec/master/examples/v2.0/json/petstore.json',
```

Comment:
Par souci de brièveté, est-ce qu'on ne peut pas remplacer ce chemin de la même manière qu'on a juste en dessous le "path/to/..." ?